packages/core/CalendarManager.ts

```diff
@@ -206,6 +206,7 @@ export const getBusyCalendarTimes = async (
     // Subtract 11 hours from the start date to avoid problems in UTC- time zones.
     const startDate = dayjs(dateFrom).subtract(11, "hours").format();
     // Add 14 hours from the start date to avoid problems in UTC+ time zones.
+    // NOTE: some APIs restrict to 31 days for fetch availability e.g. zoho calendar
```

Comment:
Is there anything else we have to do here if that's the case?

Reply:
I'm actually not sure, since this is an API limitation.

Reply:
I think we should account for this limitation in the Zoho `CalendarService` as querying for the availability in the next month throws an error. 

In the `getAvailability` method, we could split the start/end date ranges into 31 day groups and make an API call to Zoho for each group. Then combine the results and feed that back to the `CalendarManager`. If there is a way to batch calls to Zoho then that would be even better.

Reply:
Done.